Artifacts/windows-domain-join-new/artifact.ps1

```diff
@@ -68,10 +71,6 @@ else
 {
     $securePass = ConvertTo-SecureString $DomainAdminPassword -AsPlainText -Force
     Write-Output "Attempting to join the domain..."
-    # Treat empty string as null to convert to optional parameter
-    if(-not $OUPath) {
-        $OUPath = $null
-    }
     Add-VmToDomain -VmName $env:COMPUTERNAME -DomainName $DomainToJoin -JoinUser $DomainAdminUsername -JoinPassword $securePass -OU $OUPath
```

Comment:
`$OUPath` doesn't exist anymore, from what I can see. What about the `-OU` parameter in this call?